junit-jupiter-engine/src/test/java/org/junit/jupiter/api/condition/EnabledIfSystemPropertyIntegrationTests.java

```diff
@@ -25,59 +30,84 @@
  */
 class EnabledIfSystemPropertyIntegrationTests {
 
-	private static final String KEY = "EnabledIfSystemPropertyTests.key";
+	private static final String KEY1 = "EnabledIfSystemPropertyTests.key1";
+	private static final String KEY2 = "EnabledIfSystemPropertyTests.key2";
 	private static final String ENIGMA = "EnabledIfSystemPropertyTests.enigma";
 	private static final String BOGUS = "EnabledIfSystemPropertyTests.bogus";
 
 	@BeforeAll
 	static void setSystemProperty() {
```

Comment:
Maybe `setSystemProperties` like in the other one?

Reply:
Good catch. I'll change that, too.